Engine/ScriptAnalyzer.cs

```diff
@@ -1141,7 +1141,7 @@ private List<ExternalRule> GetExternalRule(string[] moduleNames)
                             {
                                 dynamic description = helpContent[0].Properties["Description"];
 
-                                if (null != description && null != description.Value && description.Value.GetType().IsArray)
+                                if (description != null && description.Value != null && description.Value.GetType().IsArray)
```

Comment:
This  is just to highlight why I needed the NuGet package for `Microsoft.CSharp`, which was included in `netstandard1.6` but not `netstandard2.0`

Reply:
yup - I found this as well when I was looking at building against PowerShellStandard.Library